code/script.js

```diff
@@ -4,16 +4,82 @@
 // Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+
 )
 
+
+let userName = prompt("Enter your name here")
+let chosenName = userName;
+alert ("Hello" + " " + chosenName + "!")
+
 // Step 2 - Food choice
 // Your code goes here
 
+let choice = prompt (`What food are we getting today? 
+Please choose 
+1 for Pizza,
+2 for Salad
+3 for Pasta` )
+switch (choice){
+  case "1":
+    alert("You chose Pizza");
+    break;
+    case "2":
+      alert("You chose Salad");
+      break;
+      case "3":
+        alert("You chose Pasta");
+        break;
+        default:
+          alert("Invalid choice. Please chose 1, 2 or 3")
+        
+}
 // Step 3 - Subtype choice
 // Your code goes here
 
+if (choice === "1"){
+  let pizzaSubchoice = prompt ("Choose extra: Please choose Small, Medium or Grande")
```

Comment:
little tip: see here the subChoice is not used for any further logic, the user can type 1 or Small but the value is not relevant for the next selection, this part has room for further improvements ✨